ext/standard/tests/array/array_first_last.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+array_first()/array_last()
+--FILE--
+<?php
+$str = "hello world";
+
+$test_cases = [
+    ["single element"],
+    [&$str, 1],
+    [1, &$str],
```

Comment:
Should a test case be added that verifies the deref?

Reply:
No, the VM will emit an assertion failure if it were to return a ref because it verifies the arginfo, which would be failing the test